01-getting-started/installation/platforms/install-on-gcp.md

```diff
@@ -0,0 +1,195 @@
+
+![Create Cloud Run Step 1](/images/started/gcp/2022-10-20_13-29.png)
+
+4. Scroll down and select **Allow all traffic** (7) and **Allow unauthenticated invocations** (8). Then expand (9) the _Containers, Connections, Security_ section. 
+
+![Create Cloud Run Step 1](/images/started/gcp/2022-10-20_13-31.png)
+
+5. Set **Container port** to _5000_ (10) and set Memory to _1_ and CPU to _1_ (11).
+
+![Create Cloud Run Step 1](/images/started/gcp/2022-10-20_13-34.png)
+
+6. Finally we set the environment variables. Click **+Add Variable** (12) and set the following environment variables (13) one by one replacing the values with your actual values where applicable, click **Create** (14) when done.
+
+| Key                                        | Value                                               | Notes                              |
+| ------------------------------------------ | --------------------------------------------------- | ---------------------------------- |
+| `[ASSETSTORE__TYPE]`                       | GoogleCloud                                         | Do not change this value           |
+| `[ASSETSTORE__GOOGLECLOUD__BUCKET]`        | _[STORAGE BUCKET NAME]_                             | Provide your own value             |
+| `[EVENTSTORE__MONGODB__CONFIGURATION]`     | _[MONGODB CONNECTION STRING]_                       | Provide your own value             |
+| `[STORE__MONGODB__CONFIGURATION]`          | _[MONGODB CONNECTION STRING]_                       | Provide your own value             |
+| `[ASPNETCORE_URLS]`                        | http://+:5000                                       | Do not change this value           |
+
+![Create Cloud Run Step 1](/images/started/gcp/2022-10-20_13-42.png)
+
+7. Wait until the deployment is complete and you have sucessful notification with a green check. Copy the URL (15) and click **Edit & Deploy New Version** (16). 
+
+![Create Cloud Run Step 1](/images/started/gcp/2022-10-20_13-52.png)
+
+8. Add a new environment variable by clicking **+Add Variable** (17) and then adding the following (18). Click **Deploy** (19) when done.
+
+| Key                                        | Value                                               | Notes                              |
+| ------------------------------------------ | --------------------------------------------------- | ---------------------------------- |
+| `[URLS__BASEURL]`                          | _[CLOUD RUN URL]_                                   | Provide your own value             |
+
+![Create Cloud Run Step 1](/images/started/gcp/2022-10-20_13-56.png)
+
+9. This should deploy a new revision of the Cloud Run and update the Base URL value in the app.
+
+![Create Cloud Run Step 1](/images/started/gcp/2022-10-20_16-45.png)
+
+10. You can now open the URL in a browser and continue with Squidex setup.
```

Comment:
A section about custom domain would be good.

Reply:
I added this in Step 8.